### ISSUE TITLE
Use the global nuget packages

### ISSUE THREAD

**Issue by jwChung** [hedgehogqa/fsharp-hedgehog#166]
Since the new project format is used, the global nuget packages are referenced as default settings. As   the `storage: none` line is set on the top in the `paket.dependencies` file, the line is not applied to each group - Build, Test.

I don't know it is intentional.  Can the line be set for each group?

**Comment by moodmosaic** [comment]
I'm afraid I don't fully understand 😕 The new *.fsproj* file format still uses [Paket](https://github.com/fsprojects/Paket).

**Comment by jwChung** [comment]
The `storage: none` line can be set for all  groups [to use the global shared directory](https://fsprojects.github.io/Paket/dependencies-file.html#Disable-packages-folder)( see the line below, indicated with "<---"). This means that the line in the Main group doesn't affect other groups.

If we set the line for the Build and Test groups, the local packages directory will not be necessary.

```
redirects: on
storage: none

// Note: it is important that we use groups to separate
// build & test dependencies, or else the test dependencies (Unquote)
// force the versions of FSharp.Core to be higher.

group Build

    storage: none <------------------------
    source https://www.nuget.org/api/v2/

    // We want to depend on FSharp.Core 4.1.17 (it's the earliest one
    // that supports .NET standard), but not require that version in
    // the generated nuget package (see paket.template).
```

**Comment by jwChung** [comment]
Nope. If so, the packages directory in the root directory of this repository  will be deleted, and all the packages which are not in the global shared directory will created newly.  All the .net solutions will share the packages in the global shared directory.

**Comment by moodmosaic** [comment]
Feel free to open a pull request if there's something significant that can be improved.

**Comment by jwChung** [comment]
Sure. I thought there might be a reason I don't catch.

**Comment by jwChung** [comment]
This issue has beed addressed in the PR(#168). I close this.
